packages/client/src/StreamVideoClient.ts

```diff
@@ -126,30 +129,30 @@ export class StreamVideoClient {
    * @param token a token or a function that returns a token.
    */
   async connectUser(
-    user?: User,
-    token?: TokenOrProvider,
+    user: User,
+    token: TokenOrProvider,
```

Comment:
Should we store these values on the client instance?
```
this.user = user;
this.token = token;
```

Reply:
We need to have `user` and `token` as params in case someone does `connectUser` separate from `constructor`.  I think the API is cleaner if they are required than if they are optional.

Reply:
yes, that's fine. What I was suggesting is, we don't seem to update the instance variables:
```ts
const client = new StreamVideoClient({ user: 'oliver' });
client.connectUser('zita');
console.log(client.user): // oliver
```

Reply:
I removed the instance variables since your comment, we don't need them anymore